### ISSUE TITLE
[bug] Deletes videos from playlist only from second time

### ISSUE THREAD

**Issue by teod0r** [TeamNewPipe/NewPipe#2355]
Press "delete" from context menu of playlist.
Exit the app.
Restart the app.
Go to the playlist and you'll see deleted video there which will be deleted only from the second time.

**Comment by teod0r** [comment]
I mean bookmarked (saved) playlists

**Comment by teod0r** [comment]
actualy it deletes a video from playlist only after I press hw BACK button hiding the application. if I long press hw BACK button for CLOSING the app after deleting a video, video stays undeleted.

**Comment by Stypox** [comment]
Duplicate of #2088
